### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by lukefisklennon** [dbankier/TiShadow#552]
I’ve read the README a few times but I’m still confused about how it all works. I started by installing `tishadow` via npm, and configuring it to use the `appc` cli (which is now free).

I started with the “dummy’s path” which right away doesn’t work, because I don’t have `titanium` installed. Is there also hook installed for `appc`? And does it work on Windows 10? (I’m using PowerShell). Running `apc run -p android -T device --shadow` doesn’t seem to do anything different from usual.

Next section, “TiShadow App”. I ran the commands in the section and it just left the directory blank. I’m also confused about this because I want to use it in my existing project, not make a new one.

I got the TiShadow server running but it’s useless without the other components.

The section “Start the TiShadow App” refers to the directory before, which is still blank, and uses a command which I still don’t have.

The next part is “What you can do with TiShadow”, subheading “Full Application Deployment”. I ran `tishadow run` (also tried adding `-p android -T device`), but it just says “44 file(s) bundled”; doesn’t build, install or launch on my device.

Then I got to “TiShadow Appify”, which I’m really confused about. How does that differ from `app`? Is it for making a new project, or generating from an existing one? Where do I run it? What do I use it for?

Basically, I don’t have a clue how it all works after reading the documentation several times. All I want is live reloads on my device as I make changes, there’s nothing particularly special that I need.

Anyway, thanks for being patient with me!

**Comment by lukefisklennon** [comment]
Update: I installed `titanium` and now it’s working fine. I think the documentation could be a little clearer about using with `appc`, but no worries. Sorry about bothering you!
